src/main/java/uk/gov/hmcts/reform/civil/enums/hearing/HearingDuration.java

```diff
@@ -0,0 +1,10 @@
+package uk.gov.hmcts.reform.civil.enums.hearing;
+
+public enum HearingDuration {
```

Comment:
could we not reuse the already existing enum?

Reply:
Created a new JIRA task for refactoring https://tools.hmcts.net/jira/browse/CIV-6823